src/components/NavbarAccount.vue

```diff
@@ -67,6 +72,8 @@ watch(
       :profile="profile"
       @close="modalAccountOpen = false"
       @login="handleLogin"
+      @open-terms="modalTermsOpen = true"
     />
   </teleport>
+  <ModalSnapshotTerms :open="modalTermsOpen" @close="modalTermsOpen = false" />
```

Comment:
Maybe we should reopen this Account modal after closing the ToS modal ?

Reply:
Also, on terms modal opening, keyboard focus remains on the `ModalAccount` modal, meaning that `TAB` key does not work, and `ESC` key is closing the ModalAccount below, instead of the terms modal